docs: add sidecar build for release blog and doc updates (#4499)
aws/copilot-cli: site/content/blogs/release-v126.en.md

## `storage init` for environment addons

## Sidecar image build

> You can now build Sidecar container images natively from a Dockerfile. Copilot builds and pushes multiple sidecar container images in parallel
> to the same ECR repository of your main container.
> You can modify your workload manifest as below.

Reviewer: ```suggestion
You can now use Copilot to build sidecar container images natively from a Dockerfile, and get them pushed to ECR.
```

Let's not mention parallel building/pushing until the PR gets merged.
You can modify your workload manifest as below.
